asana/asana.go

```diff
@@ -31,20 +31,14 @@ var defaultOptFields = map[string][]string{
 }
 
 var (
-	// ErrBadRequest can be returned on any call on response status code 400.
-	ErrBadRequest = errors.New("asana: bad request")
-	// ErrUnauthorized can be returned on any call on response status code 401.
-	ErrUnauthorized = errors.New("asana: unauthorized")
-	// ErrPaymentRequired can be returned on any call on response status code 402.
-	ErrPaymentRequired = errors.New("asana: payment required")
-	// ErrForbidden can be returned on any call on response status code 403.
-	ErrForbidden = errors.New("asana: forbidden")
-	// ErrNotFound can be returned on any call on response status code 404.
-	ErrNotFound = errors.New("asana: not found")
-	// ErrThrottled can be returned on any call on response status code 429.
-	ErrThrottled = errors.New("asana: too many requests")
-	// ErrInternal can be returned on any call on response status code 500.
-	ErrInternal = errors.New("asana: internal server error")
+	// Constants for common error message text
+	TextBadRequest      = "asana: bad request"
```

Comment:
Do these actually need to be exported? Do you want consumers to depend on the string content of the error message? Consider returning the http status code as well and keeping these strings internal.